integration_tests/mockApis/sendLegalMail.ts

```diff
@@ -351,11 +351,10 @@ const stubCreateContact = (): SuperAgentRequest =>
         'Content-Type': 'application/json;charset=UTF-8',
       },
       jsonBody: {
-        id: 1,
-        prisonerName: `{{jsonPath request.body '$.prisonerName'}}`,
-        prisonId: `{{jsonPath request.body '$.prisonId'}}`,
-        prisonNumber: `{{jsonPath request.body '$.prisonNumber'}}`,
-        prisonerDob: `{{jsonPath request.body '$.prisonerDob'}}`,
```

Comment:
This wasn't working! 😨 And no amount of hacking has fixed, so hardcoding required.